lib/y-richtext.coffee

```diff
@@ -146,7 +149,10 @@ class YRichText extends BaseClass
         author = event.changedBy
         word = event.object.val(author)
```

Comment:
please don't use `word` as a variable. this is misleading. You could use `ref` (because it's a reference).